tests/dummy/app/routes/docs/utilities/index.js

```diff
@@ -1,7 +1,10 @@
 import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
 
 export default class DocsUtilitiesIndexRoute extends Route {
+  @service router;
+
   redirect() {
-    this.replaceWith("docs.utilities.flex");
+    this.router.replaceWith("docs.utilities.confirm");
```

Comment:
The redirect is such that it redirects to the first subpage when entering the index route?